src/Template/Form.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Template;
```

Comment:
I would create Form namespace for Form classes / interfaces. We will need some Form, Field, Container, Group etc. also Field and Container are “Controls” (can be added to Form) then Form, Container and Group are “ControlHolders” (contains Controls). So Forms have complex structure and it should not be in Template namespace directly

Reply:
Would you reffer to keep Form in Tempalte os tol level element together with other top elements or move it too to Template\Form\Form?

Reply:
I think Template\Form\Form would be better to keep all form items together

Reply:
ok

Reply:
Same for LatteContext/CollectedData I expect

Reply:
Yes